modules/ppcp-wc-gateway/src/WCGatewayModule.php

```diff
@@ -132,6 +132,17 @@ static function () use ( $c ) {
 				$endpoint->handle_request();
 			}
 		);
+
+		add_action(
+			'woocommerce_paypal_payments_gateway_migrate',
+			static function () use ( $c ) {
+				$settings = $c->get( 'wcgateway.settings' );
```

Comment:
I would type hint here to help psalm and IDE.

    assert( $settings instanceof Settings );

Reply:
Done! :)